webhooks-extension/config/sink-kservice.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: webhooks-extension-sink
+  labels:
+    app: webhooks-extension-sink
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: "DOCKER_REPO/extension-sink:latest"
+            imagePullPolicy: Always
+            ports:
+            - containerPort: 8080
+            livenessProbe:
+              httpGet:
+                path: /liveness
+            readinessProbe:
+              httpGet:
+                path: /readiness
+            env:
+            - name: PORT
+              value: "8080"
+            - name: DOCKER_REGISTRY_LOCATION
+              value: DOCKER_REPO
+            - name: INSTALL_NAMESPACE
+              value: INSTALL_NAMESPACE
```

Comment:
rather than having to substitute ... you can actually declare this env var as follows:

```
- name: INSTALL_NAMESPACE
  valueFrom:
    fieldRef:
      fieldPath: metadata.namespace
```

this sets the INSTALL_NAMESPACE from the metadata